### PR TITLE
Updated dependencies

### PR DESCRIPTION
**Pull request by romatr**

### CODE REVIEW

File: service_logging.gemspec [hunk 1]
```diff
@@ -19,13 +19,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_development_dependency "bundler", "~> 1.12"
-  spec.add_development_dependency "rake", "~> 10.0"
+  spec.add_development_dependency "bundler", "~> 1.14"
```
Comment (romatr) [inline service_logging.gemspec]: SemaphoreCI doesn't support version higher than `1.14.6` 🤔

Comment (romatr) [inline service_logging.gemspec]: `¯\_(ツ)_/¯`

Comment (timurkhafizov) [inline service_logging.gemspec]: Why do we need to rely on Semaphore?

Comment (timurkhafizov) [inline service_logging.gemspec]: Why do we need to rely on Semaphore?

Comment (timurkhafizov) [reply service_logging.gemspec]: I'd say let's drop Semaphore integration since we do not use it, and move to Solano

Comment (romatr) [reply service_logging.gemspec]: @timurkhafizov Removed from Sempa[phoeCI and added into Solano.

**Pull request merged**